### DISCUSSION TITLE
Man fleeing Wiltshire crash scene attacked by emus

### DISCUSSION THREAD

**Discussion by rondlg** [fieldmuseum/EMu-Documentation#623] [EMu in the News]
https://www.bbc.com/news/uk-england-wiltshire-62312529

**Comment by fmjjones** [comment]
Emmanuel Don't do it.

[https://www.tiktok.com/@tyla/video/7122022699999743238](https://www.tiktok.com/@tyla/video/7122022699999743238 )
